soccer/src/soccer/planning/primitives/create_path.hpp

```diff
@@ -24,4 +26,16 @@ Trajectory simple(
     const MotionConstraints& motion_constraints, RJ::Time start_time,
     const std::vector<rj_geometry::Point>& intermediate_points = {});
 
+Trajectory intermediate(const LinearMotionInstant& start, const LinearMotionInstant& goal,
+                        const MotionConstraints& motion_constraints, RJ::Time start_time,
+                        const rj_geometry::ShapeSet& static_obstacles);
+
+std::vector<rj_geometry::Point> get_intermediates(const LinearMotionInstant& start,
+                                                  const LinearMotionInstant& goal);
+
+const double MIN_SCALE = 0.5, MAX_SCALE = 1.5;
```

Comment:
let's use `constexpr` here and for the other variables below because we already know the values